api/graphite.go

```diff
@@ -581,13 +580,16 @@ func (s *Server) metricsDeleteRemote(ctx context.Context, orgId uint32, query st
 
 // executePlan looks up the needed data, retrieves it, and then invokes the processing
 // note if you do something like sum(foo.*) and all of those metrics happen to be on another node,
-// we will collect all the indidividual series from the peer, and then sum here. that could be optimized
+// we will collect all the individual series from the peer, and then sum here. that could be optimized
 func (s *Server) executePlan(ctx context.Context, orgId uint32, plan expr.Plan) ([]models.Series, error) {
 
+	preExecutePlan := time.Now()
 	minFrom := uint32(math.MaxUint32)
 	var maxTo uint32
 	var reqs []models.Req
+	stats := make(map[string]float64)
```

Comment:
can you make this a struct, so we can have well defined properties and don't need additional hashmap overhead?